k8s/README.md

```diff
@@ -108,3 +108,31 @@ We'd love to hear your feedback! Subscribe to [users@streampipes.apache.org](mai
 
 ## License
 [Apache License 2.0](../LICENSE)
+
+## Troubleshooting
+### Some of the Deployments Never Reach 'Ready' State
+#### ui
+The UI Deployment specifies so-called init containers which wait for the following deployments:
+* backend
+* connect-master
+* consul
+* couchdb
+* activemq
+
+This means that when at least one of these deployments are not reachable the containers running the UI will not start up. Chances are when you fix the issues with the the others it will start up fine.
```

Comment:
```suggestion
This means that when at least one of these deployments are not reachable the containers running the UI will not start up. Chances are when you fix the issues with the others it will start up fine.
```